packages/cli/src/services/dynamic-loader.ts

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export function dynamicLoad(module: string): any {
+  return require(module)
+}
```

Comment:
I'd do this with `unknown` to force the caller to specify the type
```suggestion
export function dynamicLoad(module: string): unknown {
  return require(module)
}
```